package.json

```diff
@@ -36,8 +36,12 @@
     "test": "ember test",
     "test:all": "ember try:each"
   },
+  "peerDependencies": {
+    "ember-source": "*"
```

Comment:
I think we should probably actually include our "real" minimum version here, what do you think?

If we worry that this might be considered "breaking" to folks (I personally don't, but I could imagine someone using Ember 3.7 or something and things _happen_ to work), we can always make an issue to track updating this to an actual value at our next major release.